airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -157,8 +157,8 @@ class ClusterGenerator:
     # pylint: disable=too-many-arguments,too-many-locals
     def __init__(
         self,
-        project_id: Optional[str] = None,
-        cluster_name: Optional[str] = None,
+        project_id: str,
+        cluster_name: str,
```

Comment:
It depends if this parameter was actually optional before. I think cluster_name was a required parameter, but we had the wrong method signature.  In that case, this change doesn't change anything, so it's not a breaking change.

Reply:
Yes, this change in fact fixes the cluster generator